napari/_vispy/camera.py

```diff
@@ -3,6 +3,9 @@
 
 from napari._vispy.utils.quaternion import quaternion2euler_degrees
 
+VISPY_DEFAULT_ORIENTATION_2D = ('right', 'up', 'towards')  # xyz, not zyx
```

Comment:
I would remove the not zyx or give a bit more context.

Reply:
Ok I just came to this line confused because further down we have `DEFAULT_ORIENTATION=('towards', 'down', 'right')` and in the example we have `('up', 'right')`, so I agree we need a more expressive comment here that our default axis ordering is flipped.